ayon_api/server_api.py

```diff
@@ -1973,6 +1976,9 @@ def get_rest_projects(self, active=True, library=None):
                 yield project
 
     def get_rest_entity_by_id(self, project_name, entity_type, entity_id):
+        if not project_name or not entity_type or not entity_id:
```

Comment:
This could also be as:
`if not (project_name and entity_type and entity_id)`

Reply:
```suggestion
        if not all((project_name, entity_type, entity_id)):
```